python/cudf/tests/test_dataframe.py

```diff
@@ -1110,3 +1110,13 @@ def test_dataframe_boolean_mask_Series(gdf):
     assert gdf_masked.shape[0] == 2
     assert gdf_masked2.shape[0] == 4
     assert gdf_masked3.shape[0] == 8
+
+
+def test_iter(pdf, gdf):
+    assert list(pdf) == list(gdf)
+
+
+def test_iteritems(gdf):
+    for k, v in gdf.iteritems():
+        assert k in gdf.columns
+        assert isinstance(v, gd.Series)
```

Comment:
Shouldn't we check that we actually get `gdf[k]` and not just that `v` is a `Series`? Might be overkill though, so I'd be glad to hear your opinion.

Reply:
Overkill or not, it's easy enough to just add :)